feature-wallet-api/src/main/java/jp/co/soramitsu/feature_wallet_api/domain/model/Operation.kt

```diff
@@ -0,0 +1,119 @@
+package jp.co.soramitsu.feature_wallet_api.domain.model
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import jp.co.soramitsu.feature_wallet_api.R
+import jp.co.soramitsu.feature_wallet_api.presentation.formatters.formatTokenAmount
+import java.math.BigDecimal
+
+data class Operation(
+    val hash: String,
+    val address: String,
+    val accountName: String?,
+    val operation: Operation,
+    val time: Long,
+    val tokenType: Token.Type,
+    val nextPageCursor: String? = null
```

Comment:
As far as I remember, we agreed on saving cursor globally, not in each element in the list